### PR TITLE
Ellipse input and label text

### PR DESCRIPTION
**Pull request by jordanjones243**
# Alaska Airlines Pull Request

Please include a summary of the change and which issue is fixed. Please also include relevant motivation and context. List any dependencies that are required for this change.

**Fixes:** #186, #187, #195

## Summary:

_Please summarize the scope of the changes you have submitted, what the intent of the work is and anything that describes the before/after state of the project._

Ellipse the text of the label and input when the characters don't fit in the width of the `auro-input` element.

## Type of change:

Please delete options that are not relevant.

- [x] New capability
- [x] Revision of an existing capability
- [ ] Infrastructure change (automation, etc.)
- [ ] Other (please elaborate)

## Checklist:

- [x] My update follows the CONTRIBUTING guidelines of this project
- [x] I have performed a self-review of my own update

**By submitting this Pull Request, I confirm that my contribution is made under the terms of the Apache 2.0 license.**

_Pull Requests will be evaluated by their quality of update and whether it is consistent with the goals and values of this project. Any submission is to be considered a conversation between the submitter and the maintainers of this project and may require changes to your submission._

**Thank you for your submission!**<br>
-- Auro Design System Team

### CODE REVIEW

File: src/style.scss [hunk 1]
```diff
@@ -64,15 +64,23 @@ input {
   }
 }
 
+// .inputElement:focus-within {
+//   padding-right: 1.875rem;
+// }
+
```
Comment (jason-capsule42) [inline src/style.scss:70]:
```suggestion
// .inputElement:focus-within {
//   padding-right: 1.875rem;
// }

```

Remove commented out code

File: src/style.scss [hunk 2]
```diff
@@ -64,15 +64,23 @@ input {
...
 .inputElement {
   position: relative;
   width: 100%;
 
   padding: var(--auro-size-xl);
-  padding-right: 1.875rem;
+  padding-right: var(--auro-size-xs);
```
Comment (jason-capsule42) [inline src/style.scss:76]: Something is off with the size. The clickable area for the clear button still overlaps the input. There should be some padding in between the end of the input text and the clickable button element.

**Review by jason-capsule42 — CHANGES_REQUESTED** [review]

File: src/style.scss (hunk 2 above)
Comment (jordanjones243) [reply src/style.scss:76]: In what case are you referring to in which the clear button element is overlapping the input? When we do not have focus in the input element? I was under the impression that you told me to use all of the remaining white space when the icons were not visually present.

Comment (jason-capsule42) [reply src/style.scss:72]: When the clear icon is displayed you can interact with the DOM explorer and see that the clickable area of the clear button overlaps the input element.

**Comment by blackfalcon** [comment]
#186 Confirmed

Screen Shot 2022-10-10 at 7 30 06 PM

**Comment by blackfalcon** [comment]
#187 Confirmed

Screen Shot 2022-10-10 at 7 34 09 PM

**Comment by blackfalcon** [comment]
#195  Confirmed

Oct-10-2022 19-37-29

**Review by blackfalcon — CHANGES_REQUESTED** [review]
```
Auto-merging src/style.scss
CONFLICT (content): Merge conflict in src/style.scss
error: could not apply 63806ec... refactor(size): use new size tokens with fallback option #186
```

**Comment by blackfalcon** [comment]
Once conflicts have been resolved, feel free to release.

**Comment by jason-capsule42** [comment]
Resolve the merge conflict, releasing per review comment from @blackfalcon.

**Pull request merged**